web/packages/teleport/src/services/websession/websession.ts

```diff
@@ -190,14 +195,23 @@ const session = {
   },
 
   _fetchStatus() {
-    api.get(cfg.api.userStatusPath).catch(err => {
+    this.validateCookieAndSession().catch(err => {
       // this indicates that session is no longer valid (caused by server restarts or updates)
       if (err.response.status == 403) {
         this.logout();
       }
     });
   },
 
+  /**
+   * validateCookieAndSessionFromBackend makes an authenticated request
+   * which just checks if the cookie sent from browser is valid and user
+   * session is still valid.
```

Comment:
I'd shorten it a bit:
```suggestion
   * which checks if the cookie and the user session are still valid.
```